### PR TITLE
Show previews of ANFPad in the readme file

### LINKED ISSUE #17: [DOCUMENTATION] Show previews of ANFPad in the readme file

### PR DESCRIPTION
**Pull request by mellobacon**
Addressing #17.
Some preview images showing the different themes and the markdown renderer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
 <h3>I want to compile my own version</h3>
 <p>Well, you're in luck, there are two ways instructed in this file:</p>
-<h4><strong>[RECOMMENED]</strong> Using PowerShell</h4>
+<h4><strong>[RECOMMENDED]</strong> Using PowerShell</h4>
```
Comment (anf) [inline README.md:28]: Ayy, thanks for pointing this out and fixing it.

File: README.md [hunk 2]
```diff
@@ -64,6 +64,18 @@ npm run start;
...
+    <h5>Light Theme</h5>
+    <img width="500" alt="Light Theme" src="https://user-images.githubusercontent.com/42365887/103587106-21013200-4eac-11eb-9074-b057da4ae19d.png"></img>
+    <h5>Dark Theme</h5>
+    <img width="500" alt="Dark Theme" src="https://user-images.githubusercontent.com/42365887/103587109-21013200-4eac-11eb-92ad-030d94ea5569.png"></img>
+    <h5>Monokai Theme</h5>
+    <img width="500" alt="Monokai Theme" src="https://user-images.githubusercontent.com/42365887/103587098-1e064180-4eac-11eb-89b1-db164061c77a.png"></img>
+    <h5>Markdown Renderer</h5>
+    <img width="500" alt="Markdown Renderer" src="https://user-images.githubusercontent.com/42365887/103587101-1e9ed800-4eac-11eb-92b8-18d426b45519.png"></img>
+    </span>
+
```
Comment (anf) [inline README.md]: Requires some indentation.

**Review by anf — CHANGES_REQUESTED** [review]
You could work on what I specified in those comments.

**Comment by mellobacon** [comment]
Alrighty then

**Comment by mellobacon** [comment]
Formatted.

**Review by anf — APPROVED** [review]
Alright, looks good enough to me.

**Pull request merged**